src/libs/elektra/kdb.c

```diff
@@ -526,7 +527,39 @@ static int elektraGetDoUpdate (Split * split, Key * parentKey)
 	return 0;
 }
 
+static int elektraGetDoUpdate2 (Split * split, KeySet * ks, Key * parentKey)
```

Comment:
I do not like the duplication of the code, can you add the NR_OF_PLUGINS as parameter?